ml-agents/mlagents/trainers/demo_loader.py

```diff
@@ -0,0 +1,144 @@
+import json
+import numpy as np
+import pathlib
+import logging
+from PIL import Image
+import io
+from mlagents.trainers.buffer import Buffer
+from mlagents.envs.brain import BrainParameters, BrainInfo
+from mlagents.envs.utilities import process_pixels
+from mlagents.envs.communicator_objects import *
+from google.protobuf.internal.decoder import _DecodeVarint32
+
+logger = logging.getLogger("mlagents.trainers")
+
+
```

Comment:
Should these line in a class called `DemonstrationLoader` and be made static methods rather than having unstructured methods ?

Reply:
@eshvk Suggested they be methods not connected to a class.